test/controllers/homes_controller_test.rb

```diff
@@ -38,7 +30,7 @@ class HomesControllerTest < ActionDispatch::IntegrationTest
     assert_redirected_to home_url(@home)
   end
 
-  test "should destroy home" do
+  test "should destroy home and associated foster_homes" do
```

Comment:
Is there a way to also test that the foster_homes have been cleared out as well? Same question for a similar test in `fosters_controller_test.rb`